### PR TITLE
Add upgrade description for updates within Sulu 2.0

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Fixed tickets | ---
| License | MIT

#### What's in this PR?

This PR adds a description describing how to do an upgrade from any 2.0 version to anything below 3.0

#### Why?

There are a few steps required to do, so it is worth being documented.

### CODE REVIEW

File: upgrades/upgrade-2.0.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+   This step cannot be automated, because changes we are introducing here are possible BC breaks or might simply not
+   fit your project. Look at the changes in the `Skeleton repository`_ yourself and see which ones make sense in your
+   case.
+
+**3. Check the UPGRADE.md file for BC breaks**
+
+   The `UPGRADE.md`_ file in our ``sulu/sulu`` repository contains all changes breaking backwards compatibility. It
+   might be that none of the changes described in this file break your application, depending on which parts of Sulu
+   you have used. However, if somethings goes south, this file should contain an explanation what to change.
```
Comment (nnatter) [inline upgrades/upgrade-2.0.rst]: something

File: upgrades/upgrade-2.0.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   Run the following two npm commands (requires npm version 5 or higher) in the `assets/admin` folder of your project:
+
+.. code-block:: bash
+
+    $ npm install
+    $ npm run build
+
+.. note::
+
+    This is the only option you have to update the administration interface if you have implemented some custom
```
Comment (nnatter) [inline upgrades/upgrade-2.0.rst]: would omit `you have`

File: upgrades/upgrade-2.0.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+   In case you have not added any custom JavaScript to our administration interface you can alternatively also use the
+   pre-built JavaScript in our `Skeleton repository`_. We have implemented a Symfony command, that automatically checks
+   which Sulu version you are using and downloads the correct version of the JavaScript build.
+
+.. code-block:: bash
+
+    $ bin/console sulu:admin:download-build
+
+.. _Composer documentation: https://getcomposer.org/doc/articles/versions.md#writing-version-constraints
```
Comment (nnatter) [inline upgrades/upgrade-2.0.rst]: are those links intentional? 🙂

Comment (danrot) [reply upgrades/upgrade-2.0.rst]: Yes, they are hard to do by accident :see_no_evil: Why do you ask?

**Pull request merged**